migrations/sql/V2019.03.21.04.54__add_notifications_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE notification
+(
+    user_id serial PRIMARY KEY,
+    idir character varying(30) NOT NULL,
+    mine_guid uuid NOT NULL,
+
+    FOREIGN KEY (mine_guid) REFERENCES mine(mine_guid)
+);
```

Comment:
- The name of this table (and its associated model) don't really fit. The name should probably use the word 'subscription' rather than 'notification'. Something like 'mine_subscription'.
 - The PK should match the name of the table. In this case, it should be notification_id.
 - The 'idir' column should be more generic, just 'user_name' or something similar.